hikari/impl/rest.py

```diff
@@ -811,7 +815,9 @@ async def _parse_ratelimits(self, compiled_route: routes.CompiledRoute, response
         # isn't some weird edge case here somewhere in Discord's implementation.
         # We can safely retry if this happens.
         if remaining <= 0:
-            _LOGGER.warning("rate limited on bucket %s, maybe you are running more than one bot on this token?", bucket)
+            _LOGGER.warning(
+                "rate limited on bucket %s, maybe you are running more than one bot on this token?", bucket or "None"
```

Comment:
This shouldn't be logged if bucket is None

Reply:
Best to always log it. If Discord change the header or have a bug on their side where they stop sending it, both us and bot developers will be confused on why their bots are getting banned for many 429's without any logs

Reply:
That's just misleading, it only covers bucket ratelimits and to say we were unexpected ratelimited on an endpoint where we can't pre-emptively ratelimit is just wrong. For the cases you just described it'd raise a ratelimited error

Reply:
That or None is still redundant though

Reply:
It has already been removed. This comment is outdated